### PR TITLE
feat: add kratos boilerplate behind feature flag

### PR DESCRIPTION
**Pull request by bodymindarts**
Here is some more boilerplate on how to pass the `flowData` that we get from Kratos into the view (works both SSR and DOM rendered).

Its exposed behind a feature flag that is off by default. This is not intended to be functional at this point - but I'd like to merge / get feedback on the basic structure.

If this pattern is alright I can continue by populating the register view form and creating the other flows.

### CODE REVIEW

File: src/components/root-component.tsx [hunk 1]
```diff
@@ -1,40 +1,64 @@
...
+        <Suspense
+          fallback={
+            <div className="suspense-fallback">
+              <Spinner size="big" />
+            </div>
+          }
+        >
+          <ErrorBoundary FallbackComponent={ErrorFallback}>
+            <div id="main-container">
+              <Component flowData={flowData} />
```
Comment (samerbuna) [inline src/components/root-component.tsx:53]:
```suggestion
              <Component flowData={flowData} {...props} />
```

File: src/server/routes.ts [hunk 2]
```diff
@@ -47,12 +56,30 @@ export const checkRoute = (path: string): RoutePath | Error => {
...
-  RoutePath,
+import Register from "../components/pages/register"
+
+const authRoutesDef = {
+  "/register/email": {
+    component: Register,
+    title: "Register with Email",
+  },
+}
+
```
Comment (samerbuna) [inline src/server/routes.ts:67]: What's the reason for separating app routes from auth routes?

**Review by samerbuna — CHANGES_REQUESTED** [review]
Looks good, besides that one change.

File: src/server/routes.ts (hunk 2 above)
Comment (bodymindarts) [reply src/server/routes.ts:67]: 2 main reasons:
- easier segregation behind feature flag
- likely the whole auth code will eventually get extracted out of the web wallet code and this should make it easier to separate.

**Pull request merged**